org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/publisher/SpecMapManager.java

```diff
@@ -183,7 +183,11 @@ public String getPath(String url, String def, String rt, String id) throws FHIRE
     }
     if (special != null) {
       switch (special) {
-      case Simplifier: return "https://simplifier.net/resolve?scope="+pi.name()+"@"+pi.version()+"&canonical="+url;
+      case Simplifier: if (url.contains("fhir.philips.com")) {
```

Comment:
`pi.license()` is available at this point!

Reply:
better deal with this on line 428... (before Simplifier):

```
} else if (pi.license().equals(anObject:"not-open-source") ) {
      res.special = SpecialPackageType.NotOpenSource;
    }

Reply:
Sure, that is option 1. We should in this case probably choose a license from https://spdx.org/licenses/ ?

Reply:
yes, we do in our sushi-config: `not-open-source`

> license: not-open-source # https://www.hl7.org/fhir/valueset-spdx-license.html

Reply:
For the package.json (what publisher uses) the documentation states:
license - optional. Follow the [[spdx naming convention](https://spdx.org/licenses/)]

Reply:
OK. https://www.hl7.org/fhir/valueset-spdx-license.html is an expansion of https://www.hl7.org/fhir/valueset-spdx-license.html
I guess the [documentation](https://confluence.hl7.org/display/FHIR/NPM+Package+Specification) should be adjusted